app/views/shelters/show.html.erb

```diff
@@ -4,3 +4,7 @@
 <h3>State: <%= @shelter.state %></h3>
 <h3>Zip: <%= @shelter.zip %></h3>
 <h3><a href="/shelters/<%= @shelter.id %>/edit">Update Shelter</h3>
+<%= link_to "Delete Shelter", "/shelters/#{@shelter.id}", :method => :delete %>
```

Comment:
Google suggested `a href` can only send get requests, so went with link_to in order to specify method.